### ISSUE TITLE
Entry addition bug

### ISSUE THREAD

**Issue by Steveg297** [geoffhumphrey/brewcompetitiononlineentry#763]
When all entries are assigned and checked as being paid/received, I updated the entries and one of the entries that was already assigned a judging number pops up with the number 000000 and the paid/received buttons unchecked.  This continues to happen.  I created a test entry and assigned it the judging number 000000 and that fixed the system from arbitrarily assigning an entry to this number.  However, after checking off paid/received, saving it and going back to sort on paid or received selections, the system keeps un-checking at least one entry.

**Comment by geoffhumphrey** [comment]
Thanks for reporting, @Steveg297. I'm looking into causes, but for now, once your entry window closes (but before you sort), you can utilize one of the _regenerate judging number_ functions available from the Admin Actions drop-down on the Admin > Entries and Participants > Manage Entries screen.

**Comment by Steveg297** [comment]
Hi Geoff,
I like your software, the format is pretty good and I have some suggestions to make it better.  However, as I stated in entry #763 there are some issues with the data.  I would like you to take a look at our competition data and see what you can do to fix it.  The website is located at: 2017 Annual South Shore Brewoff I can set you up with an administrator account, if you don't already have a back door, so you can take a look at the issues we are experiencing.  Unfortunately, if you can't fix it, I'll have to look for other software to use for our next competition.  Data integrity beats any other functionality.  
  
|  
|   |  
2017 Annual South Shore Brewoff
   |  |

  |

 

Thanks,
SteveG

**Comment by geoffhumphrey** [comment]
Steve, I looked at the entry data and, yes, there is an entry called "Test" that has a judging number of 000000. I've tried to recreate the issue on my test servers without luck and have got it on my to do list to track down a possible cause to fix in the next version. 

Other than that one entry, it appears that all of the entry and user data is fine and intact. In fact, it looks to me as if you utilized the package from beginning to end as intended, as evidenced by the presence of tables, assigned judges to those tables, defined flights, scores, BOS, and listing of all winners on the home page. 

I'm not sure what data you want me to "fix" beyond looking into the single 000000 anomaly. What other issues did you experience?

**Comment by Steveg297** [comment]
Hi Geoff,

I created the test entry in order to trick the software so it wouldn't re-number another entry, like it kept doing. 

I deleted the test entry then sorted on Paid entries, there was one entry not paid, I checked it off, saved it, then resorted on Paid and it took a previous entry, changed the judging number to 000000 and deselected the paid and received statuses. 

This is what was happening during the comp when I was trying to account for all of the entries and the payments.  Adding the test entry with 000000 as the judging number forced the errors in Paix and Rec'd to occur to the test entry, not the other entries.

There were a number of people having a problem with the Paid selection being unchecked even though they had paid through PayPal, thus may be related.

Check it out again and you'll see what I mean, I deleted the Test entry.

Thanks,

SteveG
